pypika/terms.py

```diff
@@ -1060,6 +1061,25 @@ def tables_(self):
         return self.term.tables_
 
 
+class CustomFunction():
+    def __init__(self, term, params=None):
+        self.term = term
```

Comment:
Would rename term to `name`, since the verbatim term is already used to refer to objects that represent parts of SQL expressions

Reply:
Nice catch. I'll change it